<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormWarnInformation</class>
 <widget class="QWidget" name="FormWarnInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
	background-color: rgba(16, 57, 70, 1);
	border: 0px solid rgb(111, 255, 255);
	margin-top:3.3ex;
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QGroupBox::title{
	subcontrol-origin: margin;
	subcontrol-position: top center;
	background-color: rgba(19, 67, 79, 1);
	padding: 7 10000px;
}

QLineEdit{
	background-color: rgba(255, 255, 255,0.2);
	border-width:0;
	border-style:outset;
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QLabel{
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}


QRadioButton{
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}


QPushButton{
	border: 2px solid rgba(61, 255, 255,0.4);
	background-color: rgba(12, 34, 45, 0.6);
	font: 10pt &quot;微软雅黑&quot;;
	color: rgba(102, 249, 247, 1);
	text-align：center;
}

QPushButton::disabled{
	border: 0px;
	color: rgba(102, 249, 247, 0.3);
	text-align：center;
}

QPushButton::pressed{
	border: 2px solid rgba(61, 255, 255,0.8);
	background-color: rgba(12, 34, 45, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QPushButton::hover{
	border: 2.5px solid rgba(61, 255, 255,0.8);
	background-color: rgba(12, 34, 45, 0.6);
	font: 75 12pt &quot;微软雅黑&quot;;
}

QDoubleSpinBox{
	border: 0px solid rgba(61, 255, 255,0.4);
	background-color: rgba(255, 255, 255, 0.2);
	font: 12pt &quot;Times New Roman&quot;;
	color: rgba(102, 249, 247, 1);
	text-align：center;
	padding-left:10px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_realtimeWarn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>实时预警</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_HisWarn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>历史预警</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetSearch" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>起始检索时间：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="WarndateEditStart"/>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>结束检索时间：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="WarndateEditEnd"/>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string> 历史预警工况名称：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="EdWarnSearchRecord">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	background-color: rgba(255, 255, 255,1);
	color: rgb(0, 0, 0);
	border-width:0;
	border-style:outset;
	font: 12pt &quot;微软雅黑&quot;;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnWarnSearch">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string> 检索</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidgetWarn">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255,0.2);</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="BtnClearCurAllWarnRecords">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>38</height>
             </size>
            </property>
            <property name="text">
             <string> 清除当前所有历史记录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnClearCurSelectWarnRecords">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>38</height>
             </size>
            </property>
            <property name="text">
             <string> 清除当前选中记录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnClearCurWarnPageRecords">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>38</height>
             </size>
            </property>
            <property name="text">
             <string> 清除当前页面记录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>共</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_DMS_TableControl_TotalWarnNum">
            <property name="minimumSize">
             <size>
              <width>15</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>1000</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>条记录  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_18">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 25 14pt &quot;微软雅黑 Light&quot;;</string>
            </property>
            <property name="text">
             <string>每页</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="CbPageWarnRecordsNumber">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">width: 30px; height: 30px;
font: 25 14pt &quot;微软雅黑 Light&quot;;</string>
            </property>
            <property name="currentText">
             <string>1</string>
            </property>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>11</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>13</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>35</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>40</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_19">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 25 14pt &quot;微软雅黑 Light&quot;;</string>
            </property>
            <property name="text">
             <string>条    </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_20">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 25 14pt &quot;微软雅黑 Light&quot;;</string>
            </property>
            <property name="text">
             <string>第</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxWarnPageNum">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>34</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑 Light</family>
              <pointsize>14</pointsize>
              <weight>3</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 25 14pt &quot;微软雅黑 Light&quot;;</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_21">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 25 14pt &quot;微软雅黑 Light&quot;;</string>
            </property>
            <property name="text">
             <string>页  共</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LbTotalWarnPageNum">
            <property name="minimumSize">
             <size>
              <width>15</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>1000</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>页</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnWarnTable_FirstPage">
            <property name="minimumSize">
             <size>
              <width>95</width>
              <height>38</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>首页</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnWarnTable_PreviousPage">
            <property name="minimumSize">
             <size>
              <width>95</width>
              <height>38</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>上一页</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnNextWarnPage">
            <property name="minimumSize">
             <size>
              <width>95</width>
              <height>38</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>下一页</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnLastWarnPage">
            <property name="minimumSize">
             <size>
              <width>95</width>
              <height>38</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>尾页</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
